### PR TITLE
TandemFoil: T_max=10 — lr sweep (2e-4, 1e-4)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis

The golden config (T_max=10, Lion lr=3e-4) achieved val=75.59 in 14 epochs (PR #2490). With T_max=10, the LR cycles extremely rapidly (~75 full cosine cycles per epoch). In this regime, a lower LR may improve stability: the learning rate stays near zero for longer fractions of each cycle, allowing the optimizer to settle more precisely into minima rather than bouncing around them.

Lion optimizer responds differently to LR scaling than AdamW — its effective step size is bounded by the sign of the gradient, making it less sensitive to absolute LR magnitude. Testing lr=2e-4 and lr=1e-4 explores whether the optimal LR shifts down with ultra-rapid cycling.

## Instructions

Run **two separate training runs** using the golden config with only the LR changed. Use `--wandb_group tandem-tmax10-lr-sweep` so both runs are grouped in W&B.

**Run 1 — lr=2e-4:**
```bash
cd target/icml2026 && python train.py \
  --dataset tandemfoil \
  --optimizer lion \
  --lr 2e-4 \
  --cosine_t_max 10 \
  --no-use-ema \
  --model_slices 64 \
  --enable-fourier \
  --enable-te-coord-frame \
  --enable-cp-panel \
  --enable-cp-panel-tandem-only \
  --asinh-pressure \
  --residual-prediction \
  --enable-pressure-prior-addition \
  --epochs 999 \
  --wandb_group tandem-tmax10-lr-sweep
```

**Run 2 — lr=1e-4:**
```bash
cd target/icml2026 && python train.py \
  --dataset tandemfoil \
  --optimizer lion \
  --lr 1e-4 \
  --cosine_t_max 10 \
  --no-use-ema \
  --model_slices 64 \
  --enable-fourier \
  --enable-te-coord-frame \
  --enable-cp-panel \
  --enable-cp-panel-tandem-only \
  --asinh-pressure \
  --residual-prediction \
  --enable-pressure-prior-addition \
  --epochs 999 \
  --wandb_group tandem-tmax10-lr-sweep
```

Report `val_primary/surface_pressure_mae` for each run at every epoch. Note whether lower LR improves or degrades vs the 3e-4 baseline.

## Baseline

- **Primary metric:** `val_primary/surface_pressure_mae` (lower is better)
- **Current best:** 75.59 (val) / 72.12 (test) — PR #2490 (T_max=10, Lion lr=3e-4, 14 epochs, 30-min budget)
- **Baseline W&B run:** 77yoba65
- **Target:** val < 75.59
- **Reproduce baseline:** `cd target/icml2026 && python train.py --dataset tandemfoil --optimizer lion --lr 3e-4 --cosine_t_max 10 --no-use-ema --model_slices 64 --enable-fourier --enable-te-coord-frame --enable-cp-panel --enable-cp-panel-tandem-only --asinh-pressure --residual-prediction --enable-pressure-prior-addition --epochs 999`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2557 because Superseded: tetsuo reassigned to #2610 (TandemFoil lr=2e-4).
